### DISCUSSION TITLE
v4.11.0-beta.1 packages are here

### DISCUSSION THREAD

**Discussion by vnbaaij** [microsoft/fluentui-blazor#3011] [Announcements]
~We are finalizing the first beta release of the v4.11.0 packages.~ Available on NuGet now There are quite a few changes in this release. The most important ones are the changes to the DataGrid (see #2667) and a new method of packaging Icons and Emoji in separate DLL's per variant/category.

If you are using our beta or preview packages (see [docs/using-latest-daily.md](https://github.com/microsoft/fluentui-blazor/blob/dev/docs/using-latest-daily.md)), you will run into some issues already with using Icons and Emoji. Please refer to https://preview.fluentui-blazor.net/UpgradeGuide **for our updated documentation**

**We changed the feed for our preview packages. If you are already using the feed, please update your url to what is shown in the doc mentioned above. You will not receive updates otherwise**

### Why beta packages? 

As said, there are quite a few big changes implemented in this release. In some cases these could be breaking changes for your application. We would like you to try the beta packages and provide as much feedback as possible so we can make sure the final release will be as polished as possible. Our current plan is to take about 2 weeks for this.

**Comment by MarvinKlein1508** [comment]
I will give it a try next week and let you know my thoughs! :)

**Reply by vnbaaij** [reply]
We have not yet released/uploaded the beta packages! Will update the post once we have done so.

**Reply by MarvinKlein1508** [reply]
We've tested it today. It just works.

**Reply by vnbaaij** [reply]
That is GREAT to hear. Thanks!

**Comment by PascalVorwerk** [comment]
Hi! Just upgraded to the prerelease, but i am still experiencing some issues with my remote data and the autofit feature. Currently a table of 3 columns is generating the following html: 
<img width="745" alt="image" src="https://github.com/user-attachments/assets/7c37cada-0949-4d4c-8935-831fc6edcfb5" />


The first column can contain pretty large strings, but the others shouldn't. Could it be because i am not using a fluent grid system but the data grid is in a FluentStack? I'll try to reproduce the problem in a sample soon..

**Reply by PascalVorwerk** [reply]
Another small issue i just found is when using the ColumnOptions feature of a Property column. In the previous version, the column header would contain the following css:

<img width="542" alt="image" src="https://github.com/user-attachments/assets/625791d4-d893-4822-a39b-92edb23e570b" />


However, when upgrading to the new prerelease, the header is not a flexbox anymore for example. This causes the options buttons to appear above the title.  This is the css generated with the upgraded prerelease package:

<img width="698" alt="image" src="https://github.com/user-attachments/assets/6ee1f84a-4622-46a7-8b37-d1f06a3bf009" />
 
 Now, i checked the preview examples website. It does not show this error because for some reason this styling issue does not happen if the options button is added in the 2nd or later column i suppose? However, if i manually add a button to the html with the same styling to the first column (i simply copy pasted the options button into the first column header), i get the same behaviour. 
 
 
<img width="1676" alt="image" src="https://github.com/user-attachments/assets/11387e1e-fec0-4d31-bb80-faafc8a06448" />

**Reply by vnbaaij** [reply]
Thanks for reporting. Will take a look!

**Reply by vnbaaij** [reply]
When changing the example on the code side (not in the rendered HTML) so the 1st column looks like this:
```
 <TemplateColumn Title="Rank" Sortable="true"  SortBy="@rankSort" Align="Align.Center">
     <ColumnOptions>
         <div class="search-box">
             <FluentSearch type="search" Autofocus=true @bind-Value=nameFilter @oninput="HandleCountryFilter" @bind-Value:after="HandleClear" Placeholder="Country name..." />
         </div>
     </ColumnOptions>
     <ChildContent>
     <img class="flag" src="_content/FluentUI.Demo.Shared/flags/@(context.Code).svg" alt="Flag of @(context.Code)" />
     </ChildContent>
 </TemplateColumn>
```
It renders correctly:

![image](https://github.com/user-attachments/assets/46fa562f-7613-4e1c-b267-8a6f5d1c3af3)

**Reply by vnbaaij** [reply]
> Hi! Just upgraded to the prerelease, but i am still experiencing some issues with my remote data and the autofit feature. Currently a table of 3 columns is generating the following htm:
> The first column can contain pretty large strings, but the others shouldn't. Could it be because i am not using a fluent grid system but the data grid is in a FluentStack? I'll try to reproduce the problem in a sample soon..

Would love to get some repro code for this...
